libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -10,64 +10,64 @@ extern const AP_HAL::HAL& hal;
+ # define POSCONTROL_ACC_D_DT                   0.0025f // vertical acceleration controller dt default
+ # define POSCONTROL_POS_NE_P                   1.0f    // horizontal position controller P gain default
+ # define POSCONTROL_VEL_NE_P                   1.0f    // horizontal velocity controller P gain default
+ # define POSCONTROL_VEL_NE_I                   0.5f    // horizontal velocity controller I gain default
+ # define POSCONTROL_VEL_NE_D                   0.0f    // horizontal velocity controller D gain default
+ # define POSCONTROL_VEL_NE_IMAX                10.0f   // horizontal velocity controller IMAX gain default
+ # define POSCONTROL_VEL_NE_FILT_HZ             5.0f    // horizontal velocity controller input filter
+ # define POSCONTROL_VEL_NE_FILT_D_HZ           5.0f    // horizontal velocity controller input filter for D
 #else
  // default gains for Copter / TradHeli
- # define POSCONTROL_POS_Z_P                    1.0f    // vertical position controller P gain default
- # define POSCONTROL_VEL_Z_P                    5.0f    // vertical velocity controller P gain default
- # define POSCONTROL_VEL_Z_IMAX                 1000.0f // vertical velocity controller IMAX gain default
- # define POSCONTROL_VEL_Z_FILT_HZ              5.0f    // vertical velocity controller input filter
- # define POSCONTROL_VEL_Z_FILT_D_HZ            5.0f    // vertical velocity controller input filter for D
- # define POSCONTROL_ACC_Z_P                    0.5f    // vertical acceleration controller P gain default
- # define POSCONTROL_ACC_Z_I                    1.0f    // vertical acceleration controller I gain default
- # define POSCONTROL_ACC_Z_D                    0.0f    // vertical acceleration controller D gain default
- # define POSCONTROL_ACC_Z_IMAX                 800     // vertical acceleration controller IMAX gain default
- # define POSCONTROL_ACC_Z_FILT_HZ              20.0f   // vertical acceleration controller input filter default
- # define POSCONTROL_ACC_Z_DT                   0.0025f // vertical acceleration controller dt default
- # define POSCONTROL_POS_XY_P                   1.0f    // horizontal position controller P gain default
- # define POSCONTROL_VEL_XY_P                   2.0f    // horizontal velocity controller P gain default
- # define POSCONTROL_VEL_XY_I                   1.0f    // horizontal velocity controller I gain default
- # define POSCONTROL_VEL_XY_D                   0.25f   // horizontal velocity controller D gain default
- # define POSCONTROL_VEL_XY_IMAX                1000.0f // horizontal velocity controller IMAX gain default
- # define POSCONTROL_VEL_XY_FILT_HZ             5.0f    // horizontal velocity controller input filter
- # define POSCONTROL_VEL_XY_FILT_D_HZ           5.0f    // horizontal velocity controller input filter for D
+ # define POSCONTROL_POS_D_P                    1.0f    // vertical position controller P gain default
```

Comment:
If we're going to rename these definitions maybe we should put the "D" at the front?  so POSCONTROL_D_POS_P?